client/src/views/Mapping/components/InputColumns/index.tsx

```diff
@@ -37,51 +36,30 @@ const InputColumns = ({ schema, selectedAttribute, source }: IProps) => {
         let inputColumns = data && data.inputColumns ? data.inputColumns : [];
 
         return selectedAttribute.id ? (
-          <Subscription
-            subscription={subscribeAttribute}
+          <Query
```

Comment:
why do you use Query instead of Subscription now ?

Reply:
I'm not sure why but the subscription was not working as it was supposed to: the `data` returned was always null.
Also, I don't understand why subscriptions are needed for Pyrog, I think queries can always do what we want and are easier to manage.

Reply:
Subscriptions are a nicer way to update the UI using websockets without having to re-query the same fragment but I agree that it it simpler to manage for now and doesn't cost much anyway.

Reply:
I think that queries are cached and this cache is automatically changed when performing updates on a single object (in some other cases, it can be modified by hand).
Shouldn't this be preferred over having a websocket connection? (That's what I'm doing in the v2 so if you think it isn't we will need to talk about it ^^)

Reply:
subscriptions are tricky anyway, and using the apollo cache is required whether we use subscriptions or not. I think it's always best to start with queries, so keep going! we'll talk about it in more details when i'll get back